src/v/archival/ntp_archiver_service.cc

```diff
@@ -367,7 +353,22 @@ ss::future<> ntp_archiver::upload_until_abort() {
         if (!is_synced.has_value()) {
             continue;
         }
+
         vlog(_rtclog.debug, "upload loop synced in term {}", _start_term);
```

Comment:
nit: Change this line to `vlog(_rtclog.debug, "upload loop synced, current term: {}", _parent.term());`.